cli/Tests/TuistGeneratorTests/ProjectMappers/SynthesizedResourceInterfaceProjectMapperTests.swift

```diff
@@ -546,6 +546,58 @@ final class SynthesizedResourceInterfaceProjectMapperTests: TuistUnitTestCase {
         ])
     }
 
+    func testMap_whenResourceContainsBinaryPlist() throws {
```

Comment:
I'd also test the scenario when the plist is unreadable/invalid and check we get the expected error.